### PR TITLE
feat : add wishlist feature

### PR DESCRIPTION
**Pull request by Omkar-Chaskar**
Wish list feature : 
Add to wish list and remove from wish list

### CODE REVIEW

File: src/Components/WishListProduct.js [hunk 1]
```diff
@@ -1,157 +1,72 @@
...
+                          })
+                        }
+                      />
+                    </span>
+                <div className="card__media-column card__media-column-vertical">
+                  <img
+                    className="card-img-height"
+                    src={products.productImg}
+                    alt=""
+                  />
```
Comment (GeetaPogul) [inline src/Components/WishListProduct.js:40]: use prettier formatter to remove these extra spaces

**Review by GeetaPogul — APPROVED** [review]
good to be merge

File: src/Components/WishListProduct.js [hunk 2]
```diff
@@ -1,157 +1,72 @@
...
-          </div>
-          <div className="card__action-buttons text-center">
-            <button className="button button-primary p1 btn-full btn-wishlist bolder">
-              Go To Cart
-            </button>
-          </div>
-        </div>
-      </div>
+      {wishlistState.wishlist < 1 ? (
+        <p className="p3 bold center">
```
Comment (mnath30) [inline src/Components/WishListProduct.js:15]: Good use of utility classes

**Review by mnath30 — APPROVED** [review]
Code looks good and everything is well structured.
Good use of utility classes.
Good naming conventions are followed

File: src/Components/WishListProduct.js [hunk 3]
```diff
@@ -1,157 +1,72 @@
...
+                            payload: products
+                          })
+                        }
+                      />
+                    </span>
+                <div className="card__media-column card__media-column-vertical">
+                  <img
+                    className="card-img-height"
+                    src={products.productImg}
+                    alt=""
```
Comment (manojasarsa) [inline src/Components/WishListProduct.js:39]: you can give title name as `alt` attribute

**Review by manojasarsa — APPROVED** [review]
Rest, looks good!

File: src/Components/WishListProduct.js (hunk 1 above)
Comment (Omkar-Chaskar) [reply src/Components/WishListProduct.js:40]: thanks

File: src/Components/WishListProduct.js (hunk 2 above)
Comment (Omkar-Chaskar) [reply src/Components/WishListProduct.js:15]: thanks

File: src/Components/WishListProduct.js (hunk 3 above)
Comment (Omkar-Chaskar) [reply src/Components/WishListProduct.js:39]: Thanks . In next PR i will give alt attribute "title name"

**Pull request merged**